src/renderer/src/components/song/context-menu/items/PlayNext.tsx

```diff
@@ -1,28 +1,23 @@
 import { Song } from "../../../../../../@types";
 import SongContextMenuItem from "../SongContextMenuItem";
-import { Component, createSignal, Show } from "solid-js";
+import { Component } from "solid-js";
 
 type SongPlayNextProps = {
-  path: Song["path"];
+  path: Song["path"] | undefined;
 };
 
 const PlayNext: Component<SongPlayNextProps> = (props) => {
-  const [show, setShow] = createSignal(false);
-
-  window.api.listen("queue::created", () => {
-    setShow(true);
-  });
-
-  window.api.listen("queue::destroyed", () => {
-    setShow(false);
-  });
-
   return (
-    <Show when={show()}>
-      <SongContextMenuItem onClick={() => window.api.request("queue::playNext", props.path)}>
-        Play Next
-      </SongContextMenuItem>
-    </Show>
+    <SongContextMenuItem
+      onClick={() => {
+        if (props.path !== undefined && props.path !== "") {
+          window.api.request("queue::playNext", props.path);
+        }
+      }}
+    >
+      <p>Play next</p>
+      <i class="ri-menu-add-line"></i>
```

Comment:
Please use icons from 'lucide-solid' instead.  Remix icons are to be phased out in #83

Reply:
Done. Replaced with `<ListStartIcon />`